### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by Fank**
Create a automated build, follow: https://docs.docker.com/docker-hub/builds/#create-an-automated-build
I recommend creating a organization for igniterealtime and publish it under: igniterealtime/openfire (all lowercase)

To build the image local run
```bash
docker build -t openfire .
```
If you need docker get it here [linux](https://get.docker.com/), [mac](https://store.docker.com/editions/community/docker-ce-desktop-mac).

To run openfire:    
```bash
docker run -d --name openfire -v openfire-conf:/usr/local/openfire/conf -v openfire-plugins:/usr/local/openfire/plugins -v openfire-security:/usr/local/openfire/resources/security openfire
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+RUN apt-get update -qq \
+    && apt-get install -qqy ant \
+    && ant -Dhalt.on.plugin.error=true -Dno.package=true -f build/build.xml dist.bin
+FROM openjdk:8-jre
+COPY --from=0 /target/release/openfire /usr/local/openfire
+WORKDIR /usr/local/openfire
+#RUN adduser --disabled-login --uid 999 --home /usr/local/openfire --gecos "" openfire \
+#    && chown -R openfire:openfire /usr/local/openfire
+#USER openfire
+VOLUME ["/usr/local/openfire/conf", "/usr/local/openfire/embedded-db", "/usr/local/openfire/plugins", "/usr/local/openfire/resources/security"]
```
Comment (Fank) [inline Dockerfile]: Only folders which are listed here are persistant. so please validate that these are all.

**Comment by Fank** [comment]
@dwd Guus told there are existing ideas about docker, could you share them? so i can adjust this PR?

**Comment by akrherz** [comment]
@Fank  https://github.com/surevine/openfire-docker maybe

**Comment by dwd** [comment]
I think this is fine, and I don't think it'll have any ill effect on anything else, so I'm just merging as-is and we can go from there.

**Pull request merged**